### PR TITLE
Gp/prepare release 3.2.1 address indexing

### PR DESCRIPTION
**Pull request by JackPiri**
PR for release 3.2.1 preparation (_address_indexing_).
A minor change to script release-notes.py is included.
Please note this is PR against **AddressIndexing** (not **AddressIndexing_development**).

### CODE REVIEW

File: doc/man/zen-cli.1 [hunk 1]
```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.6.
-.TH ZEN-CLI "1" "September 2022" "zen-cli v3.2.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.49.1.
+.TH ZEN-CLI "1" "January 2023" "zen-cli v3.2.1-679b76dd4" "User Commands"
 .SH NAME
-zen-cli \- manual page for zen-cli v3.2.0
+zen-cli \- manual page for zen-cli v3.2.1-679b76dd4
 .SH DESCRIPTION
-Horizen RPC client version v3.2.0
+Horizen RPC client version v3.2.1\-679b76dd4
```
Comment (cronicc) [inline doc/man/zen-cli.1]: Lets change this to `v3.2.1` without the build number.

File: doc/man/zen-tx.1 [hunk 2]
```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.6.
-.TH ZEN-TX "1" "September 2022" "zen-tx v3.2.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.49.1.
+.TH ZEN-TX "1" "January 2023" "zen-tx v3.2.1-679b76dd4" "User Commands"
 .SH NAME
-zen-tx \- manual page for zen-tx v3.2.0
+zen-tx \- manual page for zen-tx v3.2.1-679b76dd4
 .SH DESCRIPTION
-Horizen zen\-tx utility version v3.2.0
+Horizen zen\-tx utility version v3.2.1\-679b76dd4
```
Comment (cronicc) [inline doc/man/zen-tx.1]: See above.

File: doc/man/zend.1 [hunk 3]
```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.6.
-.TH ZEND "1" "September 2022" "zend v3.2.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.49.1.
+.TH ZEND "1" "January 2023" "zend v3.2.1-679b76dd4" "User Commands"
 .SH NAME
-zend \- manual page for zend v3.2.0
+zend \- manual page for zend v3.2.1-679b76dd4
 .SH DESCRIPTION
-Zen Daemon version v3.2.0
+Zen Daemon version v3.2.1\-679b76dd4
```
Comment (cronicc) [inline doc/man/zend.1]: See above.

**Review by cronicc — CHANGES_REQUESTED** [review]

File: doc/man/zen-cli.1 (hunk 1 above)
Comment (JackPiri) [reply doc/man/zen-cli.1]: Addressed with force push (for keeping history clean) in 58c1f119cd7d8eae61fe426a91a562767fd76c70.

File: doc/man/zen-tx.1 (hunk 2 above)
Comment (JackPiri) [reply doc/man/zen-tx.1]: Addressed with force push (for keeping history clean) in 58c1f119cd7d8eae61fe426a91a562767fd76c70.

File: doc/man/zend.1 (hunk 3 above)
Comment (JackPiri) [reply doc/man/zend.1]: Addressed with force push (for keeping history clean) in 58c1f119cd7d8eae61fe426a91a562767fd76c70.

**Review by ptagl — APPROVED** [review]

**Review by cronicc — APPROVED** [review]

**Review by a-petrini — APPROVED** [review]

**Pull request merged**